libkbfs/kbpki_client.go

```diff
@@ -271,6 +271,18 @@ func (k *KBPKIClient) IsTeamReader(
 	return teamInfo.Writers[uid] || teamInfo.Readers[uid], nil
 }
 
+// GetTeamRootID implements the KBPKI interface for KBPKIClient.
+func (k *KBPKIClient) GetTeamRootID(ctx context.Context, tid keybase1.TeamID) (
+	keybase1.TeamID, error) {
+	teamInfo, err := k.serviceOwner.KeybaseService().LoadTeamPlusKeys(
+		ctx, tid, UnspecifiedKeyGen, keybase1.UserVersion{},
+		keybase1.TeamRole_NONE)
+	if err != nil {
+		return keybase1.TeamID(""), err
+	}
+	return teamInfo.RootID, nil
```

Comment:
Why do you say it doesn't set it?  It's set right here: https://github.com/keybase/kbfs/pull/1099/files#diff-f385c6b3885c40be40101a1ac02efdcaR577